assembly/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Request,  Response, Headers, Fastly } from "@fastly/as-compute";
+import { Request,  Response, Headers, URL, Fastly } from "@fastly/as-compute";
```

Comment:
[Ooooooh! Aaaaaaah!](https://www.youtube.com/watch?v=V5RDdKEuPQ4&t=8s) :heart_eyes_cat:

Reply:
Hahaha! :joy: Youtube links in PR reviews are the best thank you :joy: 

And yes! Thank you for all the kind words! :smile: :tada: